### PR TITLE
[Feat] - Add Sitemap Feature for the Docs

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- This PR adds a sitemap feature for the docs to improve SEO

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/develop/rust/wasinn/mediapipe.md [hunk 1]
```diff
@@ -63,10 +60,7 @@ DetectionResult:
 
 ## Understand the code
 
-The [main.rs](https://github.com/juntao/demo-object-detection/blob/main/src/main.rs) is the complete example Rust source.
-All `mediapipe-rs` APIs follow a common pattern. A Rust struct is designed to work with a model. It contains functions
-required to pre- and post-process data for the model. For example, we can create an `detector` instance
-using the builder pattern, which can build from any "object detection" model in the Mediapipe model library.
+The [main.rs](https://github.com/juntao/demo-object-detection/blob/main/src/main.rs) is the complete example Rust source. All `mediapipe-rs` APIs follow a common pattern. A Rust struct is designed to work with a model. It contains functions required to pre- and post-process data for the model. For example, we can create an `detector` instance using the builder pattern, which can build from any "object detection" model in the Mediapipe model library.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply docs/develop/rust/wasinn/mediapipe.md:63]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: docs/develop/rust/wasinn/mediapipe.md [hunk 2]
```diff
@@ -75,17 +69,15 @@ let detector = ObjectDetectorBuilder::new()
         .build_from_buffer(model_data)?;
 ```
 
-The `detect()` function takes in an image, pre-processes it into a tensor array, runs inference on the mediapipe object detection model,
-and the post-processes the returned tensor array into a human redable format stored in the `detection_result`.
+The `detect()` function takes in an image, pre-processes it into a tensor array, runs inference on the mediapipe object detection model, and the post-processes the returned tensor array into a human redable format stored in the `detection_result`.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply docs/develop/rust/wasinn/mediapipe.md:72]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: docs/develop/rust/wasinn/mediapipe.md [hunk 3]
```diff
@@ -75,17 +69,15 @@ let detector = ObjectDetectorBuilder::new()
...
 
 ```rust
 let mut input_img = image::open(img_path)?;
 let detection_result = detector.detect(&input_img)?;
 println!("{}", detection_result);
 ```
 
-Furthermore, the `mediapipe-rs` crate provides additional utility functions to post-process the data. For example,
-the `draw_detection()` utility function draws the data in `detection_result` onto the input image.
+Furthermore, the `mediapipe-rs` crate provides additional utility functions to post-process the data. For example, the `draw_detection()` utility function draws the data in `detection_result` onto the input image.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply docs/develop/rust/wasinn/mediapipe.md:80]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: docs/start/usage/use-cases.md [hunk 4]
```diff
@@ -20,4 +20,4 @@ Featuring AOT compiler optimization, WasmEdge is one of the fastest WebAssembly
 
 - WasmEdge applications can be plugged into existing application frameworks or platforms.
 
-If you have any great ideas on WasmEdge, don't hesitate to open a GitHub issue to discuss together.
\ No newline at end of file
+If you have any great ideas on WasmEdge, don't hesitate to open a GitHub issue to discuss together.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply docs/start/usage/use-cases.md:23]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: i18n/zh/docusaurus-plugin-content-docs/current/start/usage/use-cases.md [hunk 5]
```diff
@@ -20,4 +20,4 @@ Featuring AOT compiler optimization, WasmEdge is one of the fastest WebAssembly
 
 - WasmEdge applications can be plugged into existing application frameworks or platforms.
 
-If you have any great ideas on WasmEdge, don't hesitate to open a GitHub issue to discuss together.
\ No newline at end of file
+If you have any great ideas on WasmEdge, don't hesitate to open a GitHub issue to discuss together.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply i18n/zh/docusaurus-plugin-content-docs/current/start/usage/use-cases.md:23]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md [hunk 6]
```diff
@@ -75,17 +69,15 @@ let detector = ObjectDetectorBuilder::new()
...
+The `detect()` function takes in an image, pre-processes it into a tensor array, runs inference on the mediapipe object detection model, and the post-processes the returned tensor array into a human redable format stored in the `detection_result`.
 
 ```rust
 let mut input_img = image::open(img_path)?;
 let detection_result = detector.detect(&input_img)?;
 println!("{}", detection_result);
 ```
 
-Furthermore, the `mediapipe-rs` crate provides additional utility functions to post-process the data. For example,
-the `draw_detection()` utility function draws the data in `detection_result` onto the input image.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md:88]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md [hunk 7]
```diff
@@ -75,17 +69,15 @@ let detector = ObjectDetectorBuilder::new()
         .build_from_buffer(model_data)?;
 ```
 
-The `detect()` function takes in an image, pre-processes it into a tensor array, runs inference on the mediapipe object detection model,
-and the post-processes the returned tensor array into a human redable format stored in the `detection_result`.
+The `detect()` function takes in an image, pre-processes it into a tensor array, runs inference on the mediapipe object detection model, and the post-processes the returned tensor array into a human redable format stored in the `detection_result`.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md:72]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

File: i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md [hunk 8]
```diff
@@ -4,10 +4,7 @@ sidebar_position: 1
 
 # Mediapipe solutions
 
-Mediapipe is a collection of highly popular AI models developed by Google. They focus on intelligent processing of media files
-and streams. The `mediapipe-rs` crate is a Rust library for data processing using the Mediapipe suite of models. The crate provides
-Rust APIs to pre-process the data in media files or streams, run AI model inference to analyze the data, and then post-process
-or manipulate the media data based on the AI output.
+Mediapipe is a collection of highly popular AI models developed by Google. They focus on intelligent processing of media files and streams. The `mediapipe-rs` crate is a Rust library for data processing using the Mediapipe suite of models. The crate provides Rust APIs to pre-process the data in media files or streams, run AI model inference to analyze the data, and then post-process or manipulate the media data based on the AI output.
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply i18n/zh/docusaurus-plugin-content-docs/current/develop/rust/wasinn/mediapipe.md:7]: Same for this as well, https://github.com/WasmEdge/docs/pull/170/files#r1324101048

**Comment by adithyaakrishna** [comment]
@q82419 Could you please confirm regarding this, https://github.com/WasmEdge/docs/pull/170/files#r1324101048? If its not needed, I will go ahead and remove it :)

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
> @q82419 Could you please confirm regarding this, https://github.com/WasmEdge/docs/pull/170/files#r1324101048? If its not needed, I will go ahead and remove it :)

It's not the point. I merged your PR since sitemap is an important feature.

**Comment by adithyaakrishna** [comment]
@alabulei1 Ohhh, I got it now, I guess you meant to say to remove the changes from this PR? 😅

**Pull request merged**